### PR TITLE
Upgrade libraries to their latest version

### PR DESCRIPTION
**Pull request by otrosien**
Upgrades all dependencies

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,8 +3,8 @@ plugins {
     id 'eclipse'
     id 'idea'
     id 'jacoco-report-aggregation'
-    id 'com.github.ben-manes.versions' version '0.49.0'
-    id 'org.owasp.dependencycheck' version '9.0.7'
+    id 'com.github.ben-manes.versions' version '0.51.0'
```
Comment (otrosien) [inline build.gradle:6]: https://github.com/ben-manes/gradle-versions-plugin/releases

File: build.gradle [hunk 2]
```diff
@@ -3,8 +3,8 @@ plugins {
...
+    id 'org.owasp.dependencycheck' version '9.0.9'
```
Comment (otrosien) [inline build.gradle:7]: https://github.com/dependency-check/dependency-check-gradle/releases

File: fahrschein-http-api/build.gradle [hunk 3]
```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 dependencies {
-    api "com.github.luben:zstd-jni:1.5.5-6"
+    api "com.github.luben:zstd-jni:1.5.5-11"
```
Comment (otrosien) [inline fahrschein-http-api/build.gradle:7]: https://github.com/luben/zstd-jni/compare/v1.5.5-6...v1.5.5-11

File: fahrschein-metrics-dropwizard/build.gradle [hunk 4]
```diff
@@ -5,7 +5,7 @@ plugins {
 
 dependencies {
     api project(':fahrschein')
-    api 'io.dropwizard.metrics:metrics-core:4.2.23'
+    api 'io.dropwizard.metrics:metrics-core:4.2.25'
```
Comment (otrosien) [inline fahrschein-metrics-dropwizard/build.gradle:8]: https://github.com/dropwizard/metrics/releases  4.2.25 is not tagged in github 👎

File: fahrschein-metrics-micrometer/build.gradle [hunk 5]
```diff
@@ -5,7 +5,7 @@ plugins {
 
 dependencies {
     api project(':fahrschein')
-    api 'io.micrometer:micrometer-core:1.12.1'
+    api 'io.micrometer:micrometer-core:1.12.4'
```
Comment (otrosien) [inline fahrschein-metrics-micrometer/build.gradle:8]: https://github.com/micrometer-metrics/micrometer/releases

File: fahrschein-opentelemetry/build.gradle [hunk 6]
```diff
@@ -8,7 +8,7 @@ dependencies {
     api "io.opentelemetry:opentelemetry-api:${property('opentelemetry.version')}"
     testImplementation "io.opentelemetry:opentelemetry-sdk-testing:${property('opentelemetry.version')}"
     testImplementation "io.opentelemetry:opentelemetry-extension-trace-propagators:${property('opentelemetry.version')}"
-    testImplementation("org.assertj:assertj-core:3.24.2")
+    testImplementation("org.assertj:assertj-core:3.25.3")
```
Comment (otrosien) [inline fahrschein-opentelemetry/build.gradle:11]: https://github.com/assertj/assertj/releases

File: fahrschein/build.gradle [hunk 7]
```diff
@@ -15,10 +15,10 @@ dependencies {
 
     testImplementation("org.mock-server:mockserver-junit-jupiter-no-dependencies:5.15.0")
 
-    testImplementation ('com.jayway.jsonpath:json-path:2.8.0') {
+    testImplementation ('com.jayway.jsonpath:json-path:2.9.0') {
```
Comment (otrosien) [inline fahrschein/build.gradle:18]: https://github.com/json-path/JsonPath/releases

File: fahrschein/build.gradle [hunk 8]
```diff
@@ -15,10 +15,10 @@ dependencies {
...
         exclude group: "org.ow2.asm", module: "asm"
     }
-    testImplementation("ch.qos.logback:logback-classic:1.4.14") {
+    testImplementation("ch.qos.logback:logback-classic:1.5.3") {
```
Comment (otrosien) [inline fahrschein/build.gradle:21]: https://logback.qos.ch/news.html

File: gradle.properties [hunk 9]
```diff
@@ -2,17 +2,17 @@
 project.version=2.0.2
 
 ## dependency versions
-jackson.version=2.16.0
-jackson_databind.version=2.16.0
+jackson.version=2.17.0
```
Comment (otrosien) [inline gradle.properties:5]: https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.17

File: gradle.properties [hunk 10]
```diff
@@ -2,17 +2,17 @@
...
+jackson_databind.version=2.17.0
 slf4j.version=2.0.9
 spring.version=6.1.4
 springboot.version=3.2.3
-mockito.version=5.8.0
-junit.version=5.10.1
-testcontainers.version=1.19.3
+mockito.version=5.11.0
```
Comment (otrosien) [inline gradle.properties:10]: https://github.com/mockito/mockito/releases

File: gradle.properties [hunk 11]
```diff
@@ -2,17 +2,17 @@
...
+junit.version=5.10.2
+testcontainers.version=1.19.7
```
Comment (otrosien) [inline gradle.properties:12]: https://github.com/testcontainers/testcontainers-java/releases

File: gradle.properties [hunk 12]
```diff
@@ -2,17 +2,17 @@
...
 okhttp.version=4.10.0
 apachehttp.version=5.3
-opentelemetry.version=1.33.0
+opentelemetry.version=1.36.0
```
Comment (otrosien) [inline gradle.properties:15]: https://github.com/open-telemetry/opentelemetry-java/releases

File: gradle/wrapper/gradle-wrapper.properties [hunk 13]
```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
```
Comment (otrosien) [inline gradle/wrapper/gradle-wrapper.properties:3]: https://docs.gradle.org/8.6/release-notes.html

**Comment by mesut** [comment]
👍

File: fahrschein-metrics-dropwizard/build.gradle (hunk 4 above)
Comment (otrosien) [reply fahrschein-metrics-dropwizard/build.gradle]: using 4.2.24

**Comment by otrosien** [comment]
👍

**Comment by otrosien** [comment]
closing. We should not use the github.com repo anymore.

**Pull request closed without merging**